hook!-full-game/src/Player/States/Wall.gd

```diff
@@ -0,0 +1,39 @@
+extends State
+
+const WALL_SLIDE_SPEED: = 100.0
```

Comment:
It's an acceleration as you add `WALL_SLIDE_SPEED * delta` to the velocity. Same for wall pull strength

I'd have these variables as exported vars rather than constants, to make it easier to configure the state